src/components/body.tsx

```diff
@@ -0,0 +1,11 @@
+import { Sidebar } from "./sidebar";
+import { Content } from "./content";
+
+export function Body() {
+  return (
+    <div className="body">
+      <Sidebar />
+      <Content />
+    </div>
```

Comment:
Consider using semantic `<main>` element. Additionally, I would recommend rendering the sidebar at a higher level and composing the app in a slightly different way. In other words, I would imagine that `Header`, `Sidebar`, and `Content` would all essentially be "siblings" in the DOM tree.